py/torch_tensorrt/dynamo/runtime/_PythonTRTModule.py

```diff
@@ -0,0 +1,244 @@
+from typing import Any, List, Sequence
+
+# @manual=//deeplearning/trt/python:py_tensorrt
+import tensorrt as trt
+import torch
+from torch_tensorrt.fx.utils import unified_dtype_converter, Frameworks
+
+
+class TRTModule(torch.nn.Module):
```

Comment:
Lets call this PythonTorchTRT module. Basically the name of the other other one +python. Also can you note in the docstring that serializing to torchscript isnt not supported for this module?

Reply:
done